frontend/dockerfile/docs/reference.md

```diff
@@ -1617,6 +1618,37 @@ to keep the layer count in the resulting image as low as possible. Therefore,
 with the `--parents` flag, the Buildkit is capable of packing multiple
 `COPY` instructions together, keeping the directory structure intact.
 
+### COPY --exclude
+
+> **Note**
+>
+> Available in [`docker/dockerfile-upstream:master-labs`](#syntax).
+> Will be included in `docker/dockerfile:1.7-labs`.
```

Comment:
question: is this right?

Reply:
`1.7-labs` is fine when `v0.13.0` will be released. Currently we have this one available: https://github.com/moby/buildkit/releases/tag/dockerfile/1.7.0-rc1-labs